script.js

```diff
@@ -19,21 +44,32 @@ function buildGrid(size) {
 
 buildGrid(16);
 
+//
+/**
+ * Nice! You can do this in a one-liner if you want to be fancy and use more modern JS
+ *
+ * Becomes this: const getRandomColor = () => `#{Math.floor(Math.random() * 16777215).toString(16)}`;
+ *
+ * But that's kind of hard to read when you're first learning the language
+ */
 function getRandomColor() {
-    let randomColor = '#' + Math.floor(Math.random()*16777215).toString(16);
+    let randomColor = '#' + Math.floor(Math.random() * 16777215).toString(16);
     return randomColor;
 }
 
+
 function drawOnGrid() {
     let cells = document.querySelectorAll('.gridCell');
+
+    // Can you thihnk of why this may be an unsafe operation?
```

Comment:
Same as my comment @line 35?

Reply:
Same answer, except this one is potentially more risky - because we're generating `.gridCell div`s programmatically... so if `buildGrid` ever fails (somehow), there are no `div`s to grab here. A simple conditional is probably overkill... but good practice:

```js
if (cells) { ... }
```